### PR TITLE
[ADD] Add a scrollbar to the list of contacts

### PR DESCRIPTION
**Pull request by SandrinePDR**
If the list of contacts grows too long, one couldn't scroll the list. This issue is fixed with this PR.

### CODE REVIEW

File: pandora-client-web/src/components/accountContacts/accountContacts.tsx [hunk 1]
```diff
@@ -178,31 +178,33 @@ function ShowFriends() {
...
-					<th>Actions</th>
-				</tr>
-			</thead>
-			<tbody>
-				{ friendsWithStatus.map((friend) => (
-					<FriendRow key={ friend.id } { ...friend } />
-				)) }
-			</tbody>
-		</table>
+		<div style={ { overflow: 'auto', height: '100%' } }>
```
Comment (Sekkmer) [inline pandora-client-web/src/components/accountContacts/accountContacts.tsx:181]: how about using `<Scrollable direction='vertical'>` instead?

(may still need height, not sure)

**Review by Sekkmer — COMMENTED** [review]
good catch :)

thought we had an issue but I could not find any, I guess it was only mentioned on discord and we forgot to make an issue here

File: pandora-client-web/src/components/accountContacts/accountContacts.tsx (hunk 1 above)
Comment (SandrinePDR) [reply pandora-client-web/src/components/accountContacts/accountContacts.tsx]: Good idea. I didn't know that one existed...
Why does the Scrollbar on directMessages.tsx#31 not work for the contacts?

**Review by Jomshir98 — APPROVED** [review]
Looks good! Thank you.

**Pull request merged**